Cargo.toml

```diff
@@ -4,13 +4,21 @@ version = "0.1.0"
 authors = ["Laurence Tratt <laurie@tratt.net>"]
 edition = "2018"
 
+[build-dependencies]
+cfgrammar = "0.5"
+lrlex = "0.5"
+lrpar = "0.5"
+
 [dependencies]
+cfgrammar = "0.5"
```

Comment:
Is it correct that `cfgrammar` is both a build and a run dependency?

Reply:
Yes.